mass-deployer/pkg/mass-deployer/deployer.go

```diff
@@ -20,8 +20,10 @@ import (
 	"github.com/threefoldtech/zos/pkg/gridtypes"
 )
 
-const maxDeploymentRetries = 5
-const maxGoroutinesToFetchState = 100
+const (
+	maxDeploymentRetries      = 5
```

Comment:
rename to DefaultMaxRetries and use that everywhere when needed if not overridden